.github/workflows/build-base-images.yml

```diff
@@ -84,6 +89,7 @@ jobs:
     name: Generate SBOM
     runs-on: ubuntu-22.04
     needs: [build-image]
+    if: github.event_name == 'pull_request'
```

Comment:
we need to have up to date SBOMs for every build. The problem is that dependencies are changing quite often since we are using plain ubuntu underneath. That's why we need this generation also on the weekly builds...

Reply:
or if we want to work with releases now, we could also disable the scheduled build...

Reply:
OK. Re-enabled.